### DISCUSSION TITLE
Which variable storse the p value created by tcga_surv_get() and tcga_surv_plot()?

### DISCUSSION THREAD

**Discussion by ghost** [openbiox/UCSCXenaShiny#316] [Q&A] [answered]
Good morning everyone!

I hope you are all doing well.

As indicated by the title, I'm currently trying to figure out where the p-value displayed in the TCGA: suvival analysis Kaplan-Meier plot is stored. I'm in the process of generating numerous results and subsequently I'd like to filter those with statistical significance. To achieve this, I need help to find where/how I can find the statistical significance of each output from the tcga_surv_get() / tcga_surv_plot() function(s)

**Comment by ShixiangWang** [comment, marked as answer]
I recommend you getting the data based on `tcga_surv_get()`, then run batch Cox analysis with [ezcox](https://github.com/ShixiangWang/ezcox)?

**Reply by ShixiangWang** [reply]
tcga_surv_plot did not store the p-value, it only stores the ggplot object from survminer package.

**Reply by ghost** [reply]
This clarifies it, thank you so much for the help.
